src/components/atoms/Timeline/index.tsx

```diff
@@ -0,0 +1,245 @@
+              left: currentPosition + PADDING_HORIZONTAL - knobSize / 2,
+            }}>
+            {renderKnob()}
+          </Icon>
+        </MemoryBox>
+      </Container>
+    );
+  },
+  (prev, next) => prev.currentTime === next.currentTime,
+);
+
+const MONTH_LABEL_LIST = [
+  "Jan",
+  "Feb",
+  "Mar",
+  "Apr",
+  "May",
+  "Jun",
+  "Jul",
+  "Aug",
+  "Sep",
+  "Oct",
+  "Nov",
+  "Dec",
+];
+
+type MemoryListProps = {
+  start: Date;
+  memoryCount: number;
+  hoursCount: number;
+};
+
+const MemoryList: React.FC<MemoryListProps> = memo(
+  function MemoryListPresenter({ start, memoryCount, hoursCount }) {
+    return (
+      <MemoryContainer>
+        {[...Array(memoryCount + 1)].map((_, idx) => {
+          const isHour = idx % hoursCount === 0;
+          const isStrongMemory = idx % (hoursCount * STRONG_MEMORY_HOURS) === 0;
+          if (isStrongMemory) {
+            const d = new Date(start.getTime() + idx * EPOCH_SEC * MEMORY_INTERVAL);
+            const year = d.getFullYear();
+            const month = MONTH_LABEL_LIST[d.getMonth()];
+            const date = `${d.getDate()}`.padStart(2, "0");
+            const hour = `${d.getHours()}`.padStart(2, "0");
+            // TODO: Fix date format
+            const label = `${month} ${date} ${year} ${hour}:00:00.0000`;
+
+            return (
+              <LabeledMemory>
+                <MemoryLabel>{label}</MemoryLabel>
+                <Memory key={`${idx}`} isHour={isHour} isStrongMemory={isStrongMemory} />
+              </LabeledMemory>
+            );
+          }
+          return <Memory key={`${idx}`} isHour={isHour} isStrongMemory={isStrongMemory} />;
+        })}
+      </MemoryContainer>
+    );
+  },
+  (prev, next) => prev.memoryCount === next.memoryCount && prev.start === next.start,
+);
+
+const Container = styled.div`
+  background: ${({ theme }) => theme.main.lighterBg};
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: 5px 12px;
+  box-sizing: border-box;
+`;
+
+const MemoryBox = styled.div`
+  border: 1px solid #b5b5b5;
+  border-radius: 5px;
+  box-sizing: border-box;
+  position: relative;
+  overflow-x: auto;
+  overflow-y: hidden;
+  width: 100%;
+  ::-webkit-scrollbar {
+    height: 5px;
+  }
+  ::-webkit-scrollbar-track {
+    background-color: transparent;
+  }
+  ::-webkit-scrollbar-thumb {
+    border-radius: 5px;
+    background-color: #aaa;
+  }
+`;
+
+const Icon = styled.div`
+  position: absolute;
+  top: 2px;
+  color: #00bebe;
+`;
+
+const MemoryContainer = styled.div`
+  display: flex;
+  gap: 0 ${GAP_HORIZONTAL}px;
+  min-width: 100%;
+  height: 30px;
+  align-items: flex-end;
+  padding-left: ${PADDING_HORIZONTAL}px;
+  ::after {
+    content: "";
+    display: block;
+    padding-right: ${PADDING_HORIZONTAL}px;
+    height: 1px;
+  }
+`;
+
+const LabeledMemory = styled.div`
+  display: flex;
+  align-items: flex-end;
+  position: relative;
+  height: 100%;
+`;
+
+const MemoryLabel = styled.span`
```

Comment:
All text in Re:Earth should make use of the `Text` atom component.

Reply:
I fixed by https://github.com/reearth/reearth-web/pull/256/commits/ac9b61cab488a1111456d3ff6ccdfd318ee89d0b.